tgui/packages/tgui/interfaces/common/AccessList.js

```diff
@@ -0,0 +1,132 @@
+import { sortBy } from 'common/collections';
+import { Fragment } from 'inferno';
+import { useLocalState } from '../../backend';
+import { Button, Flex, Grid, Section, Tabs } from '../../components';
+
+const diffMap = {
+  0: {
+    icon: 'times-circle',
+    color: 'bad',
+  },
+  1: {
+    icon: 'stop-circle',
+    color: null,
+  },
+  2: {
+    icon: 'check-circle',
+    color: 'good',
+  },
+};
+
+export const AccessList = (props, context) => {
```

Comment:
Is this for the ID console? 🤔

Reply:
looks like it

Reply:
Given that that's not wired up yet, I would remove this file until we have our own version. It's somewhat misleading, and we can always refer to the upstream version if needed.